frontend/src/app/home/home.component.ts

```diff
@@ -11,12 +12,17 @@ export class HomeComponent implements OnInit {
 
   public restaurants: Restaurant[] = [];
 
-  constructor(private restaurantsService: RestaurantsService) {
+  constructor(private restaurantsService: RestaurantsService, private locationService: GeolocationService) {
   }
 
   ngOnInit() {
-    this.restaurantsService.getNearbyRestaurants()
-      .then(restaurants => this.restaurants = restaurants);
+
+      let promise = this.locationService.getCurrentPosition();
+      promise.then((crd) => {
+        alert("New coordinates youssef are: " + crd.latitude + " and " + crd.longitude);
```

Comment:
`alert("New coordinates youssef are: " + crd.latitude + " and " + crd.longitude);`
hahhaha

Reply:
Goood work @NaitYoussef  ❤️

Reply:
hahaha I'm going to remove it, thanks 😉